windows-nsis/openvpn.nsi

```diff
@@ -244,7 +250,11 @@ Section /o "${PACKAGE_NAME} Service" SecService
 
 	SetOutPath "$INSTDIR\bin"
 	File /oname=openvpnserv2.exe "${OPENVPNSERV2_EXECUTABLE}"
-	File "${OPENVPN_ROOT}\bin\openvpnserv.exe"
+        ${If} ${RunningX64}
+                File "${OPENVPN_ROOT_X86_64}\bin\openvpnserv.exe"
+        ${Else}
+                File "${OPENVPN_ROOT_I686}\bin\openvpnserv.exe"
+        ${EndIf}
```

Comment:
Here are spaces instead of tabs. We could add a client-side [Git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) to prevent spaces for the openvpn-build project. Or maybe we could make Travis fail if there are spaces in certain files such as this one?